src/Validator/UserRequestValidator.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Validator;
+
+use Codeception\Util\HttpCode;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+use App\Repository\UsersRepository;
+
+class UserRequestValidator
+{
+
+    private $usersrepository;
+    public function __construct(UsersRepository $usersrepository)
+    {
+        $this->usersrepository = $usersrepository;
+    }
+
+    public function isJsonRequestTypeValid(Request $request): bool
+    {
+
+        if ($request->getContentType() == 'json') {
+            return true;
+        } else {
+            throw new HttpException(HttpCode::BAD_REQUEST, "Wrog data format");
+        }
+    }
+
+    public function isJsonParamsValid(array $params): bool
```

Comment:
this function should return void, cuz if something is invalid then exception will be thrown